vizro-core/examples/default/app_dev.py

```diff
@@ -78,43 +81,43 @@ def create_benchmark_analysis():
                     sort_action="native",
                     style_cell={"textAlign": "left"},
                 ),
-                actions=[vm.Action(function=filter_interaction(targets=["line_country"]))],
-            ),
-            vm.Graph(
-                id="line_country",
-                figure=px.line(
-                    df_concat,
-                    title="Country vs. Continent",
-                    x="year",
-                    y="gdpPercap",
-                    color="color",
-                    labels={"year": "Year", "data": "Data", "gdpPercap": "GDP per capita"},
-                    color_discrete_map={"Country": "#afe7f9", "Continent": "#003875"},
-                    markers=True,
-                    hover_name="country",
-                ),
-            ),
-            vm.Button(
-                text="Export data",
-                actions=[
-                    vm.Action(
-                        function=export_data(
-                            targets=["line_country"],
-                        )
-                    ),
-                ],
-            ),
-        ],
-        controls=[
-            vm.Filter(column="continent", selector=vm.Dropdown(value="Europe", multi=False, title="Select continent")),
-            vm.Filter(column="year", selector=vm.RangeSlider(title="Select timeframe", step=1, marks=None)),
-            vm.Parameter(
-                targets=["line_country.y"],
-                selector=vm.Dropdown(
-                    options=["lifeExp", "gdpPercap", "pop"], multi=False, value="gdpPercap", title="Choose y-axis"
-                ),
+                #     actions=[vm.Action(function=filter_interaction(targets=["line_country"]))],
             ),
+            # vm.Graph(
```

Comment:
As soon as we merge the ag-grid, I agree with AnnMarie, that we should replace the data table with the ag-grid  in our demos :)